### ISSUE TITLE
Someone messed up the French translation file

### ISSUE THREAD

**Issue by rene-coty** [GeopJr/Tuba#714]
### Describe the bug

Someone largely rewrote the fr.po file in Weblate a few weeks ago and replaced its content with another language (apparently 'occitan').
The result is that the UI of the app will appear in an unknown language for French users, when they update to version 0.6.

I already rewrote (again!) the entire translation on Weblate and it's ready to be merged with the next PR from the translation platform, or myself but I'd prefer not to short-circuit the classic process for now… anyway _the sooner the better_ (that's the point of opening this issue :)

### Steps To Reproduce

Open the app with French as locale → UI is in occitan

### Instance Backend

Mastodon

### Operating System

Fedora 39

### Package

Flatpak

**Comment by rene-coty** [comment]
I don't know what consequences should be drawn from this accident. Perhaps additional verification for new contributors or new strings added... but that would make the process more cumbersome.

**Comment by GeopJr** [comment]
Thanks for notifying me about it and fixing it :bow: I'll merge it tomorrow just in case other instances are found!

Not sure what can be done, checking every single translation is way too difficult and much work to do alone :/

**Comment by rene-coty** [comment]
Thanks for your reply! I agree that a reviewing process would be too difficult, and it's probably just an error by a translator who uploaded the wrong file 🤷🏻‍♂️

**Comment by LukaszH77** [comment]
It looks like someone put an Occitan (language used in southern France and Monaco, parts of Italy and Spain) translation in place of the French.

**Comment by Nico7as** [comment]
Hi everyone,

I might be the one responsible...
Very sorry for that!! 

I'm new to translations and to using .po files.
I started translating through Weblate, but someone showed me how to save time by using POEdit and a .tmx  file.
So I ended up downloading, then uploading the file.

I'll try to be more cautious, since I might be the main occitan translator for this software.

Thanks a million to @rene-coty for fixing it !
